### DISCUSSION TITLE
Daily 05/12/2023

### DISCUSSION THREAD

**Discussion by Sinan-Karakaya** [Sinan-Karakaya/R-Type#37] [Daily]
05/12/2023

**Comment by Sinan-Karakaya** [comment]
I finished the rework of the ECS (short night)
I'll keep working on the integration with the editor

**Comment by mlbonniec** [comment]
J'essaie toujours de build le projet sur le M1 🤡 
Tant que je ne réussi pas c'est compliqué d'avancer

**Comment by JasonLAGOUTE** [comment]
Je vais commencer a faire la liaison entre les infos que je vais recevoir et ce que je vais envoyer.

**Comment by lucasdpt** [comment]
Suite de la lib network

**Comment by jovanHillion** [comment]
Je bosse sur le scripting avec le nouveau ecs
